tests/network/vmi_networking.go

```diff
@@ -733,12 +740,13 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 
 				Expect(verifyClientServerConnectivity(clientVMI, serverVMI, tcpPort, k8sv1.IPv4Protocol)).To(Succeed())
 			},
-				table.Entry("with a specific port number [IPv4]", []v1.Port{{Name: "http", Port: 8080}}, ""),
-				table.Entry("without a specific port number [IPv4]", []v1.Port{}, ""),
-				table.Entry("with custom CIDR [IPv4]", []v1.Port{}, "10.10.10.0/24"),
+				table.Entry("with a specific port number [IPv4]", []v1.Port{{Name: "http", Port: 8080}}, 8080, ""),
+				table.Entry("with a specific port used by live migration", portsUsedByLiveMigration(), int(portsUsedByLiveMigration()[0].Port), ""),
```

Comment:
You could pass the port directly here. I think it would be more readable.

Reply:
Done